src/Lykke.Job.TransactionHandler/Projections/OperationHistoryProjection.cs

```diff
@@ -177,7 +176,7 @@ public class OperationHistoryProjection
 
         public async Task Handle(IssueTransactionStateSavedEvent evt)
         {
-            await _log.WriteInfoAsync(nameof(OperationHistoryProjection), nameof(IssueTransactionStateSavedEvent), evt.ToJson(), "");
+            _log.WriteInfo(nameof(IssueTransactionStateSavedEvent), evt.ToJson(), "");
```

Comment:
This is not needed with latest version of Lykke.Cqrs - such logging is done from the library